File: src/main/java/net/adventuregame/items/Gold.java
package net.adventuregame.items;

import net.adventuregame.entities.Entity;
import net.adventuregame.items.entities.ItemEntity;
import net.adventuregame.models.TexturedModel;
import org.joml.Vector3f;

public class Gold extends Item {
    public Gold() {
        super("Gold");
    }

    @Override
    public ItemEntity createEntity(Vector3f position) {
        return null;
    }
}

File: src/main/kotlin/net/adventuregame/items/Gold.kt
package net.adventuregame.items

import net.adventuregame.items.entities.ItemEntity
import net.adventuregame.models.TexturedModel
import org.joml.Vector3f

class Gold() : Item("Gold") {
    override fun createEntity(position: Vector3f): ItemEntity? {
        return null
    }

    override fun getGuiModel(): TexturedModel? {
        return null
    }
}
